src/processor/processor.module.ts

```diff
@@ -46,25 +48,25 @@ import { NonceService } from './nonce.service';
       inject: [ConfigService],
     }),
     BullModule.registerQueue({
-      name: 'graphUpdateQueue',
+      name: ReconnectionServiceConstants.GRAPH_UPDATE_QUEUE_NAME,
       defaultJobOptions: {
         attempts: 3,
         backoff: {
           type: 'exponential',
         },
-        removeOnComplete: false,
+        removeOnComplete: true,
```

Comment:
Just confirming, now  service is caching the graph update jobs and using them? because this will clear the job queue once they are successful i.e. tx hash has been generated.

Reply:
Not quite. See here:
https://github.com/AmplicaLabs/reconnection-service/blob/ac9fed76d2b19f605a58886bf95d675c55475365/src/processor/graph-update-queue-consumer.service.ts#L138

Once the tx hashes have been submitted/generated, we move this job to the "waiting for children" state to await successful monitoring of the transactions. Once _that_ job completes, then this job can be completed/removed automatically.